README.md

```diff
@@ -1,2 +1,18 @@
 # tidb-snowflake
 Replicate from TiDB to Snowflake
+
+## replicate incremental data from TiDB to Snowflake
+
+> **Warning**
+> We do not support ddl replication yet. Any ddl operation may cause data loss.
```

Comment:
Whether the downstream supports ddl？

Reply:
no, not support any ddl now. Should stop replication, do ddl both upstream and downstream, then restart replication.